plugins/poll/assets/javascripts/discourse/components/poll-options.gjs

```diff
@@ -13,51 +14,67 @@ export default class PollOptionsComponent extends Component {
     return this.args.votes.includes(option.id);
   };
 
+  get classes() {
+    return this.args.isIrv ? "irv-poll-options" : "";
+  }
+
   @action
   sendClick(option) {
     this.args.sendOptionSelect(option);
   }
 
+  @action
+  sendRank(option, rank = 0) {
+    this.args.sendOptionSelect(option, rank);
+  }
   <template>
-    <ul>
+    <ul class={{this.classes}}>
```

Comment:
user `{{concatClass (if @isIrv "irv-poll-options")}}` instead

Reply:
resolved, thanks for the tip:  https://github.com/discourse/discourse/pull/27155/commits/b0eb09b52121fb63c8a6e79337a25da5da56b274